.evergreen/config/functions.yml

```diff
@@ -143,6 +159,13 @@ functions:
           LOAD_BALANCER=${LOAD_BALANCER} \
           REQUIRE_API_VERSION=${REQUIRE_API_VERSION} \
           bash ${DRIVERS_TOOLS}/.evergreen/run-orchestration.sh
+    - command: shell.exec
+      params:
+        script: |
+          printf "\n" >> mo-expansion.yml
+          printf "MONGODB_VERSION: '%s'\n" "${MONGODB_VERSION}" >> mo-expansion.yml
+          printf "TOPOLOGY: '%s'\n" "${TOPOLOGY}" >> mo-expansion.yml
+          cat mo-expansion.yml
```

Comment:
Why was this necessary? `$MONGODB_VERSION` and `$TOPOLOGY` should already be available in the environment, since they're referenced in the previous `shell.exec` command. The inclusion of `cat` here also looks like it may be leftover debugging.

Based on the comment below, `expansions.update` was only necessary to add 
`MONGODB_URI` and `CRYPT_SHARED_LIB_PATH`, which are written in [`drivers_orchestration.py`](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/orchestration/drivers_orchestration.py).

Reply:
`MONGODB_VERSION` and `TOPOLOGY` are passed as variables to the `bootstrap mongo-orchestration` function only, so they are no longer available later in the run. Adding them to the expansion file created by the script allows us to reuse the values later.